tutorials/chapter1_annexB.md

```diff
@@ -0,0 +1,897 @@
+<p align="right">
+  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+</p>
+
+# Chapter 1 - Annex B: Publisher and subscriber arguments
+
+- [Example B1a: Concurrent publisher and slow listener](#example-b1a-concurrent-publisher-and-slow-listener)
+  - [Source code](#source-code)
+  - [Sample output](#sample-output)
+- [Example B1b: Publishing by reference](#example-b1b-publishing-by-reference)
+  - [Sample output](#sample-output-1)
+- [Example B1c: Adding the smart loop](#example-b1c-adding-the-smart-loop)
+  - [Sample output](#sample-output-2)
+- [Example B1d: Adding an initializer](#example-b1d-adding-an-initializer)
+  - [Sample output](#sample-output-3)
+- [Example B1e: Adding a cloner](#example-b1e-adding-a-cloner)
+  - [Sample output](#sample-output-4)
+- [Example B2a: Small pool size](#example-b2a-small-pool-size)
+  - [Source code](#source-code-1)
+  - [Sample output](#sample-output-5)
+- [Example B2b: Small buffer size](#example-b2b-small-buffer-size)
+  - [Sample output](#sample-output-6)
+- [Example B3: Data multiplexer](#example-b3-data-multiplexer)
+  - [Source code](#source-code-2)
+  - [Sample output](#sample-output-7)
+
```

Comment:
Some small comments:
- For uniformity, I suggest to do the Table of Contents as an ordered list because all the other documents will be using it (see .md files changes in the PR https://github.com/klepsydra-technologies/kpsr-tutorial/pull/35/files)
- I found that to do nested lists it is recommended to use 4 spaces (e.g.: https://itsfoss.com/markdown-nested-lists/ and https://stackoverflow.com/questions/37575916/how-to-markdown-nested-list-items-in-bitbucket)
- It would be nice to have an Introduction section